Tests/TensorFlowTests/LossTests.swift

```diff
@@ -216,6 +216,34 @@ final class LossTests: XCTestCase {
             [-0.0625, -0.01490036, 0.04759964, 0.0]])
         assertEqual(computedGradient, expectedGradient, accuracy: 1e-6)
     }
+    
```

Comment:
Minor: please remove whitespace-only lines.

```suggestion

```

If you use Xcode on macOS, there's a setting to automate this: `Xcode > Preferences > Text Editing > ☑️ Automatically trim trailing whitespace ☑️ Including whitespace-only lines`.